docs/studio/0002-use-next.md

```diff
@@ -0,0 +1,19 @@
+# 2. Use next
+
+Date: 2024-06-27
+
+## Status
+
+Accepted
+
+## Context
+
+Isomer is undergoing a rebuild from the original application that was on express for backend + react/chakra for frontend
+
+## Decision
+
+Isomer is going to use starter-kit (prisma, next, kysely) to build our application
```

Comment:
Might be worthwhile to mention this Next.js pages router. We also made a conscious decision not to go with RSC for reasons I have outlined [here](https://opengovproducts.slack.com/archives/CK68JNFHR/p1716775203875659)

Reply:
added a line